### PR TITLE
chore(local debug): prepare local prerequisites cmd

### PR DESCRIPTION
**Pull request by swatDong**
Prepare for new local prerequisites cmd - new `fx-extension.validate-local-prerequisites` vscode command.

### CODE REVIEW

**Comment by swatDong** [comment]
(Not target on Jan release)
@1openwindow @HuihuiWu-Microsoft @nliu-ms - please help review the `extension` and `handler` changes due to new vsc command.

**Review by dooriya — APPROVED** [review]

File: packages/vscode-extension/package.json [hunk 1]
```diff
@@ -490,6 +490,11 @@
         "title": "Teams - Validate SPFx Dependencies",
         "enablement": "never"
       },
+      {
+        "command": "fx-extension.validate-local-prerequisites",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/package.json:494]: Why add this command in package.json if it'll never show up on UI?

Comment (swatDong) [reply packages/vscode-extension/package.json:494]: It will be only used by `tasks.json`, to replace `fx-extension.validate-dependencies`

**Review by HuihuiWu-Microsoft — DISMISSED** [review]
LGTM

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    try {
+      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DebugPrerequisites, {
+        [TelemetryProperty.Success]: TelemetrySuccess.Yes,
+      });
+    } catch {
+      // ignore telemetry error
+    }
+  } catch (error: any) {
+    try {
+      ExtTelemetry.sendTelemetryErrorEvent(TelemetryEvent.DebugPrerequisites, error as FxError);
```
[the remark this answers is not in the document]
Comment (swatDong) [reply packages/vscode-extension/src/debug/prerequisitesHandler.ts]: Good point. Added `assembleError` to ensure fx error.

**Review by kuojianlu — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**